src/main/java/mike/view/CLIView.java

```diff
@@ -82,6 +82,12 @@ public CLIView(Model classModel) throws IOException {
 						"parameter", "relationship"),
 				new NullCompleter()
 			),
+			new ArgumentCompleter(
+				new StringsCompleter("create"),
+				new StringsCompleter("relationship"),
+				new StringsCompleter("aggregation", "composition", "inheritance", "realization"),
+				new NullCompleter()

```

Comment:
Since we're gonna use a second completer for relationships, I think we should take relationship out of the first one, and have the second one include "delete" in its list of possible things to complete too.

Reply:
Good idea, I'll push the changes shortly